disassemblers/ofrak_angr/mypy.ini

```diff
@@ -3,3 +3,18 @@ files = ofrak_angr
 
 [mypy-angr.*]
 ignore_missing_imports = True
+
+[mypy-tempfile312.*] # TODO remove the next two lines when upgrading mypy. Current version has bug.
```

Comment:
What version of mypy? What bug?

This comment isn't really useful. I'd recommend just putting a comment in this PR to document whatever this is trying to communicate and removing this comment. We should use the issue tracker to track issues, not comments.